src/spring/azext_spring/_params.py

```diff
@@ -268,6 +268,8 @@ def load_arguments(self, _):
                    help='A json file path for the persistent storages to be mounted to the app')
         c.argument('loaded_public_certificate_file', options_list=['--loaded-public-certificate-file', '-f'], type=str,
                    help='A json file path indicates the certificates which would be loaded to app')
+        c.argument('deployment_name', type=str, default='default',
+                   help='Name of the active deployment.', validator=validate_name)
```

Comment:
is this `active deployment` or `default deployment`? According to line 148 in app.py, this should be `default deployment`.

Reply:
Sure. Will change to `default deployment`